3D-microservices-observability/go-chi/README.MD

```diff
@@ -0,0 +1,38 @@
+# Wavefront Hackathon - Go App
+
+This is a sample Go application using `chi` router framework called beachshirts (#[beachops](https://medium.com/@matthewzeier/thoughts-from-an-operations-wrangler-how-we-use-alerts-to-monitor-wavefront-71329c5e57a8)) which makes cool shirts for the beach.
+
+## Running the Application Locally
+
+1. `git clone` this repo and navigate to this dir:
+    ```bash
+    git clone https://github.com/wavefrontHQ/hackathon.git
+    cd hackathon/3D-microservices-observability/go-chi/
+    ```
+2. Build the binary for services:
+    ```bash
+    go build -o target/beachshirts main/beachshirts.go
+    ```
+    **Note**: Requires Go 1.11
+
+3. Run all the services using the commands below:
+    ```bash
+    ./target/beachshirts conf/shopping.conf
```

Comment:
For simplicity, it'd be useful to have an all-in-one mode. Where running `./target/beachshirts` starts all the three services at once, and if conf is provided start a particular service.